src/index.html

```diff
@@ -1,23 +1,102 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<meta name="description" content="Update your description here" />
-	<title>Update your title here</title>
+	<title>Find your cocktail</title>
 	<!--
 		Webpack will automatically link your CSS file for you.
 		Work in the src/css/styles.scss file.
 	-->
 </head>
+
 <body>
 	<main class="content-container">
-		<h1 class="page-heading">Welcome to your final project!</h1>
+		<div class="fixed">
+			<p class="instr">Please choose your...</p>
+			<img src="https://stat.ameba.jp/user_images/20211111/08/halfkneazle/3d/9c/p/o0835062615029669704.png?caw=800"
+				alt="glass">
+		</div>
+
+		<!-- Create Form -->
+		<form id="form">
+			<div class="form-control">
+				<label class="choice">ABV </label>
+				<!-- Input Type Checkbox -->
+				<label for="inp-1">
+					<input type="checkbox" name="inp">>30%</input></label>
+				<label for="inp-2">
+					<input type="checkbox" name="inp">15%~30%</input></label>
+				<label for="inp-3">
+					<input type="checkbox" name="inp">1%~15%</input></label>
+				<label for="inp-4">
+					<input type="checkbox" name="inp">0</input></label>
+			</div>
+		</form>
+
+		<form id="form2">
+			<div class="form-control">
+				<label class="choice">BASE</label>
```

Comment:
In your last label, ABV is an abbreviation, so I think it makes sense to capitalize it in the HTML. However, if you keep this capitalized, a screen reader will read it as B A S E to the user (individual letters). I would keep this proper cased, and use CSS's `text-transform: uppercase;` to apply this style instead.